### PR TITLE
WIP: Implement display of connected devices

### LINKED ISSUE #1652: Add layout to view connected devices
**Actual Behaviour**

There is no layout for the devices section when the device is successfully connected to the server

**Expected Behaviour**

Add the layout to view connected devices.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #1652 
Steps: 
- [ ] Add rooms to devices
- [ ] Add the devices to the server
- [ ] Retrieve data from the server
- [x] Display the devices

Need to implement all the steps then only we can have this feature to show data.

Changes: 
A list of connected devices appears. The realm database is used to store the information.

Enhancement: 
Enhancement of UI and date/time device got connected needs to be stored.

### CODE REVIEW

**Comment by atm1504** [comment]
@iamareebjamal @batbrain7 @arundhati24 please review it.

File: app/src/main/res/layout/fragment_connected_device.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        android:text="@string/connected_device_status" />
+
+    <android.support.v7.widget.RecyclerView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:id="@+id/connectedDevices">
+    </android.support.v7.widget.RecyclerView>
+
+
+
```
Comment (arundhati24) [inline app/src/main/res/layout/fragment_connected_device.xml]: Extra lines.

Comment (atm1504) [reply app/src/main/res/layout/fragment_connected_device.xml]: I will surely remove it.

**Comment by atm1504** [comment]
I will soon add the feature to get data from the API endpoint and store it int the realm database and use this stored data to display. So, I am changing the label to WIP.

**Comment by mariobehling** [comment]
This PR is open for 10 days. Please do follow Best Practices and split up issues. There is no point in having a PR for such a long time.

**Comment by atm1504** [comment]
@iamareebjamal @batbrain7 @arundhati24 please review it. As the JSON response didn't provide any information about the devices connected, so we have implemented this feature using the realm database itself.

**Comment by atm1504** [comment]
After #2266

**Comment by atm1504** [comment]
@mariobehling currently these features work offline. To take data from the server and show them, we first need to implement #2266 . I have already started working on that issue from today. That would be completed in 2 to 3 days.

**Pull request closed without merging**